src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -35,7 +35,7 @@ public void connect() throws Exception {
         conn = DriverManager.getConnection("jdbc:sqlite:");
         stat = conn.createStatement();
         stat.executeUpdate(
-                "create table test (id integer primary key, fn float default 0.0, sn not null, intvalue integer(5), realvalue real(8,3));");
+                "create table test (id integer primary key, fn float default 0.0, sn not null, intvalue integer(5), realvalue real(8,3), charvalue varchar(21));");
```

Comment:
what would the added column test compared to the existing `interger(5)` one, in the context of that change?

Reply:
@gotson True. It does not add a test case for this change. All it does is test something else, which is an explicit versus implicit (`sn not null`) VARCHAR.

Reply:
No that's fine, more tests is good!